path_config.go

```diff
@@ -64,12 +68,27 @@ func pathConfig(b *azureSecretBackend) *framework.Path {
 				Type:        framework.TypeBool,
 				Description: "Enable usage of the Microsoft Graph API over the deprecated Azure AD Graph API.",
 			},
+			"default_expiration": &framework.FieldSchema{
+				Type: framework.TypeString,
+				// 28 weeks (~6 months) -> days -> hours
+				Default:     (28 * 7 * 24 * time.Hour).String(),
```

Comment:
Preferable to define this duration as a constant as it is used in multiple place. A comment about how 6 months was arrived at would also be good.